see/image_providers/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2015-2016 F-Secure
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you
+# may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.  See the License for the specific language governing
+# permissions and limitations under the License.
+
+from see.image_providers.dummy import DummyProvider
+from see.image_providers.libvirt_pool import LibvirtPoolProvider
+from see.image_providers.os_glance import GlanceProvider
```

Comment:
We'd better not add glance and keystone as SEE dependencies if not needed. Let's just make sure that when we import SEE, if we don't use the glance provider, we don't get a ImportError.